docker-compose.yml

```
@@ -38,6 +38,32 @@ services:
       interval: 5s
       retries: 20
 
+  # applies OpenFGA's database schema migrations
+  openfga-migrate:
+    depends_on:
+      postgres:
+        condition: service_healthy
+    image: openfga/openfga:latest
+    container_name: osrd-openfga-migrate
+    command: migrate
+    environment:
+      OPENFGA_DATASTORE_ENGINE: "postgres"
+      OPENFGA_DATASTORE_URI: "postgres://osrd:password@postgres:5432/osrd"
+
+  openfga:
+    depends_on:
+      openfga-migrate:
+        condition: service_completed_successfully
+    image: openfga/openfga:latest
+    container_name: osrd-openfga
+    environment:
+      OPENFGA_DATASTORE_ENGINE: "postgres"
+      OPENFGA_DATASTORE_URI: "postgres://osrd:password@postgres:5432/osrd"
+    command: run --http-addr=:8091 --playground-port=8092
+    ports:
+      - "8091:8091"
+      - "8092:8092" # openfga playground /!\ DISABLE IN PROD
```

Docker compose are not used in production
```suggestion
      - "8092:8092"
```

---

That was just a reminder, also indicating the existence of the playground ^^